README.md

```diff
@@ -2,11 +2,6 @@
 
 Reports status and metrics of Docker containers running into a host.
 
-## Disclaimer
-
-This integration is at the moment at private beta stage. Running it does not
```

Comment:
"This integration is currently in private beta. Some features may not work as expected."